src/jaxsim/api/joint.py

```diff
@@ -123,7 +123,10 @@ def position_limit(
     """
 
     if model.number_of_joints() <= 1:
-        return jnp.empty(0).astype(float), jnp.empty(0).astype(float)
+        s_min = model.kin_dyn_parameters.joint_parameters.position_limits_min
+        s_max = model.kin_dyn_parameters.joint_parameters.position_limits_max
+
+        return s_min.astype(float), s_max.astype(float)
```

Comment:
Mmh I don't remember why I used `<= 1` instead of `== 0`. At first look it seems to me that the original case was correct for models with no joints.

Furthermore, I'd apply `jnp.atleast_1d` to the returned limits.

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/275/commits/f46e61866462fe3c2be2b357545f3e358b7f97b4

Reply:
What I meant is to keep the previous logic with `jnp.empty` for jointless model, and apply `atleast_1d` to the returned limits of few lines below. Maybe it can be done in #272.

Reply:
Just to be sure:

```python
@jax.jit
def position_limit(
    model: js.model.JaxSimModel, *, joint_index: jtp.IntLike
) -> tuple[jtp.Float, jtp.Float]:
    """
    Get the position limits of a joint.

    Args:
        model: The model to consider.
        joint_index: The index of the joint.

    Returns:
        The position limits of the joint.
    """

    if model.number_of_joints() == 0:
        return jnp.empty(0).astype(float), jnp.empty(0).astype(float)

    exceptions.raise_value_error_if(
        condition=jnp.array(
            [joint_index < 0, joint_index >= model.number_of_joints()]
        ).any(),
        msg="Invalid joint index '{idx}'",
        idx=joint_index,
    )

    s_min = jnp.atleast_1d(
        model.kin_dyn_parameters.joint_parameters.position_limits_min
    )[joint_index]
    s_max = jnp.atleast_1d(
        model.kin_dyn_parameters.joint_parameters.position_limits_max
    )[joint_index]

    return s_min.astype(float), s_max.astype(float)
```

Reply:
yep something like that, thanks!